index.html

```diff
@@ -142,18 +146,35 @@ <h2>News</h2>
             <!-- <li><a href="#settingsTabs-3">Thrive Options</a></li> -->
           </ul>
           <div id="settingsTabs-1">
-            <div id="browseFilesButton" class="BottomButton" style="">
-              <p style="text-align: left;">
+            <div id="browseFilesButton" class="BottomButton" style="height: 40px;">
+              <p style="text-align: left; font-size: 0.8em;">
                 Browse Installed Files
               </p>
             </div>
+            <div id="selectInstallLocation" class="BottomButton" style="height: 40px;">
+              <p style="text-align: left; font-size: 0.8em;">
+                Select Install Location
+              </p>
+            </div>
+            <div id="resetInstallLocation" class="BottomButton" style="font-size: 1.5em; vertical-align: middle;">
+                &#x21bb
```

Comment:
I think this needs a tooltip or something. To me just the icon isn't super clear what it does.